go/vt/wrangler/reparent.go

```diff
@@ -79,38 +78,26 @@ func (wr *Wrangler) InitShardPrimary(ctx context.Context, keyspace, shard string
 func (wr *Wrangler) PlannedReparentShard(
 	ctx context.Context,
 	keyspace, shard string,
-	primaryElectTabletAlias, avoidTabletAlias *topodatapb.TabletAlias,
-	waitReplicasTimeout, tolerableReplicationLag time.Duration,
+	prsOp reparentutil.PlannedReparentOptions,
 ) (err error) {
 	_, err = reparentutil.NewPlannedReparenter(wr.ts, wr.tmc, wr.logger).ReparentShard(
 		ctx,
 		keyspace,
 		shard,
-		reparentutil.PlannedReparentOptions{
-			AvoidPrimaryAlias:   avoidTabletAlias,
-			NewPrimaryAlias:     primaryElectTabletAlias,
-			WaitReplicasTimeout: waitReplicasTimeout,
-			TolerableReplLag:    tolerableReplicationLag,
-		},
+		prsOp,
```

Comment:
minor nit: `opts` is probably the more standard name here (same below with `ersOp`)

Reply:
Yep sounds good.